### PR TITLE
feat: added labs page

### PR DESCRIPTION
**Pull request by prosazhin**
This PR adds the [Labs Page](https://deploy-preview-322--cilium.netlify.app/labs/).

### CODE REVIEW

File: content/labs/2022-11-09-isovalent-cilium-enterprise-network-policies/index.md
Comment (xmulligan) [inline content/labs/2022-11-09-isovalent-cilium-enterprise-network-policies/index.md:1]: this lab should be removed

File: content/labs/2022-11-21-isovalent-cilium-enterprise-tls-visibility/index.md
Comment (xmulligan) [inline content/labs/2022-11-21-isovalent-cilium-enterprise-tls-visibility/index.md:1]: this lab should be removed

File: content/labs/2023-02-01-cilium-enterprise-zero-trust-visibility/index.md
Comment (xmulligan) [inline content/labs/2023-02-01-cilium-enterprise-zero-trust-visibility/index.md:1]: enterprise this lab should be removed

File: content/labs/2023-07-19-isovalent-cilium-enterprise-security-visibility/index.md
Comment (xmulligan) [inline content/labs/2023-07-19-isovalent-cilium-enterprise-security-visibility/index.md:1]: enterprise this lab should be removed

File: content/labs/2023-07-20-isovalent-cilium-enterprise-connectivity-visibility/index.md
Comment (xmulligan) [inline content/labs/2023-07-20-isovalent-cilium-enterprise-connectivity-visibility/index.md:1]: enterprise this lab should be removed

File: content/labs/2022-11-21-getting-started-with-ebpf/index.md
Comment (xmulligan) [inline content/labs/2022-11-21-getting-started-with-ebpf/index.md:1]: not related to cilium, should be removed

**Comment by xmulligan** [comment]
Can you also add https://academy.solo.io/learn/courses/8/introduction-to-cilium and https://www.edx.org/course/introduction-to-cilium?

**Review by sashasushko — APPROVED** [review]
Approve for the "code part". Please figure out with other issues.

File: content/labs/2022-11-09-isovalent-cilium-enterprise-network-policies/index.md
Comment (prosazhin) [reply content/labs/2022-11-09-isovalent-cilium-enterprise-network-policies/index.md:1]: Done! Please check

File: content/labs/2022-11-21-isovalent-cilium-enterprise-tls-visibility/index.md
Comment (prosazhin) [reply content/labs/2022-11-21-isovalent-cilium-enterprise-tls-visibility/index.md:1]: Done! Please check

File: content/labs/2023-02-01-cilium-enterprise-zero-trust-visibility/index.md
Comment (prosazhin) [reply content/labs/2023-02-01-cilium-enterprise-zero-trust-visibility/index.md:1]: Done! Please check

File: content/labs/2023-07-20-isovalent-cilium-enterprise-connectivity-visibility/index.md
Comment (prosazhin) [reply content/labs/2023-07-20-isovalent-cilium-enterprise-connectivity-visibility/index.md:1]: Done! Please check

File: content/labs/2022-11-21-getting-started-with-ebpf/index.md
Comment (prosazhin) [reply content/labs/2022-11-21-getting-started-with-ebpf/index.md:1]: Done! Please check

**Comment by xmulligan** [comment]
Can you make sure the getting started labs are the first ones listed on the all labs page? Also the Linux Foundation and Solo.io Academy branding should be removed from the cards

**Comment by prosazhin** [comment]
> Can you make sure the getting started labs are the first ones listed on the all labs page? Also the Linux Foundation and Solo.io Academy branding should be removed from the cards

File: content/labs/2022-10-10-the-linux-foundation-introduction-to-cilium/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+date: '2022-11-09T11:00:00.000Z'
+externalUrl: 'https://www.edx.org/course/introduction-to-cilium'
+ogImage: cover.jpg
+categories:
+  - Getting Started
+place: Online
+title: 'The Linux Foundation: Introduction to Cilium'
```
Comment (xmulligan) [inline content/labs/2022-10-10-the-linux-foundation-introduction-to-cilium/index.md:8]:
```suggestion
title: 'Introduction to Cilium'
```

File: content/labs/2022-10-11-soloio-academy-introduction-to-cilium/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+date: '2022-11-09T11:00:00.000Z'
+externalUrl: 'https://academy.solo.io/learn/courses/8/introduction-to-cilium'
+ogImage: cover.jpg
+categories:
+  - Getting Started
+place: Online
+title: 'Solo.io Academy: Introduction to Cilium'
```
Comment (xmulligan) [inline content/labs/2022-10-11-soloio-academy-introduction-to-cilium/index.md:8]:
```suggestion
title: 'Introduction to Cilium'
```

File: content/labs/2022-10-10-the-linux-foundation-introduction-to-cilium/index.md (hunk 1 above)
Comment (prosazhin) [reply content/labs/2022-10-10-the-linux-foundation-introduction-to-cilium/index.md]: Done!

File: content/labs/2022-10-11-soloio-academy-introduction-to-cilium/index.md (hunk 2 above)
Comment (prosazhin) [reply content/labs/2022-10-11-soloio-academy-introduction-to-cilium/index.md]: Done!

File: content/labs/2022-10-10-the-linux-foundation-introduction-to-cilium/index.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+date: '2022-11-09T11:00:00.000Z'
+externalUrl: 'https://www.edx.org/course/introduction-to-cilium'
+ogImage: cover.jpg
+categories:
+  - Getting Started
+place: Online
+title: 'Introduction to Cilium'
+ogSummary: 'Get a practical introduction to using Cilium as the networking plug-in for Kubernetes, including installation, observability with Hubble, securing network connections, and multi-cluster support - all based on eBPF for scalability, performance, and security.'
+draft: false
+from: 'edX'
```
Comment (xmulligan) [inline content/labs/2022-10-10-the-linux-foundation-introduction-to-cilium/index.md:11]:
```suggestion
from: 'Linux Foundation'
```

**Comment by xmulligan** [comment]
When opening the "all labs" page, is it possible to randomize the order in which they appear each time?

File: content/labs/2022-10-10-the-linux-foundation-introduction-to-cilium/index.md (hunk 3 above)
Comment (prosazhin) [reply content/labs/2022-10-10-the-linux-foundation-introduction-to-cilium/index.md]: Done! Please check

Comment (prosazhin) [reply content/labs/2022-10-10-the-linux-foundation-introduction-to-cilium/index.md]: But it was necessary to hide the 'Online' tag to fit on a single line.

File: src/layouts/main/main.jsx [hunk 4]
```diff
@@ -24,6 +25,7 @@ const navigation = [
   {
     name: 'Learn',
     childItems: [
+      { icon: LabsIcon, name: 'Labs', href: '/labs' },
```
Comment (xmulligan) [inline src/layouts/main/main.jsx:28]:
```suggestion
      { icon: LabsIcon, name: 'Labs', href: '/labs/categories/getting-started/' },
```

File: src/components/pages/labs/banner/banner.jsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+import Button from 'components/shared/button';
+import Heading from 'components/shared/heading';
+
+const Banner = () => (
+  <div className="mt-10 flex w-full flex-col items-center justify-center space-y-6 rounded-xl bg-dark-blue px-10 py-14 md:mt-16 lg:mt-24 lg:flex-row lg:space-x-8 lg:space-y-0 lg:px-4 xl:mt-32 xl:space-x-14">
+    <Heading className="text-white" tag="h2" size="sm">
+      Want to add your lab to the list? Submit a PR here
+    </Heading>
+    <Button to="https://github.com/cilium/cilium.io/pulls" theme="primary-1" target="_blank">
```
Comment (xmulligan) [inline src/components/pages/labs/banner/banner.jsx:11]:
```suggestion
    <Button to="https://github.com/cilium/cilium.io/blob/main/CONTRIBUTING.md#listing-labs" theme="primary-1" target="_blank">
```

**Review by xmulligan — APPROVED** [review]

**Pull request merged**